bucket/dbeaver-ea.json

```diff
@@ -13,18 +13,14 @@
     "architecture": {
         "64bit": {
             "url": "https://dbeaver.io/files/ea/dbeaver-ce-25.0.1-win32.win32.x86_64.zip",
-            "hash": "53dd9037f4cfe3f899ded6ed81d39bb38768a467f4ffef24c991af5c3b23f327"
+            "hash": "d3726bcedeedc9d1601c63956d36a17fa111bef70699011a176992a6fdcba3f3"
         }
     },
     "extract_dir": "dbeaver",
     "bin": [
         [
-            "dbeaver.exe",
-            "dbeaver-cli.exe"
```

Comment:
Keey `dbeaver.exe` as `dbeaver-ea`?

Reply:
@niheaven Are you referring to the command that launches the DBeaver EA GUI (not the CLI) ?

Reply:
The original manifest is broken. Notice it's a list within a list, so it implies to create a shortcut with alias `dbeaver-cli.exe` to executable `dbeaver.exe`, which is obviously wrong.

But, yes, I can do the change as you suggest. I don't often launch GUI apps from the command line, but good idea.

Reply:
done